client/landing/jetpack-cloud/sections/auth/get-token/index.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import JetpackLogo from 'components/jetpack-logo';
+import Main from 'components/main';
+import PulsingDot from 'components/pulsing-dot';
+
+const GetToken: FunctionComponent = () => {
+	const translate = useTranslate();
+	return (
+		<Main className="get-token">
+			<JetpackLogo full monochrome={ false } size={ 72 } />
+			<p>{ translate( 'Loading user…' ) }</p>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Loading Users', { context: 'Placeholder text while fetching users.'} )` *ES Score: 8*